extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.css

```diff
@@ -70,13 +70,24 @@ amp-story-page[active] .i-amphtml-story-ad-link {
   z-index: 100001 !important;
 }
 
+[dir=rtl] .i-amphtml-ad-overlay-container {
+  left: auto !important;
+  right: 0 !important;
+}
+
 [desktop] .i-amphtml-ad-overlay-container {
   /* On desktop a story page has a with of 45vh. */
   left: calc(50vw - 22.5vh) !important;
   /* And a height of 75 vh. */
   top: 12.5vh !important;
 }
 
+[dir=rtl] [desktop] .i-amphtml-ad-overlay-container {
```

Comment:
Does this work? Should we remove the space between the two attribute selectors?

Reply:
I think it's okay for now, but technically, the `dir` attribute can be present on any element: https://www.w3.org/TR/html4/struct/dirlang.html#h-8.2

Perhaps we should call this out in the documentation that we expect it on the `<html>` element.

Reply:
SG.

Reply:
Actually AMP expects it on `<html>` or `<body>` so that should be okay.
https://github.com/ampproject/amphtml/blob/94706951a241c1d8391f7e9b56ca06a55de9cb44/src/dom.js#L706-L711